apps/passport-server/src/apis/devconnect/devconnectPretixAPI.ts

```diff
@@ -140,6 +140,23 @@ export async function getDevconnectPretixAPI(): Promise<IDevconnectPretixAPI | n
   return api;
 }
 
+export type DevconnectPretixI18nMap = { [lang: string]: string };
+export type I18nLanguage = "en" | "de" | "cn"; // @todo which other languages are supported?
```

Comment:
I think we should just iterate over all of the languages until we find one that exists, starting with english. in the case that none exists, throw an error.